channels/us.m3u

```diff
@@ -1,8 +1,6 @@
 #EXTM3U
 #EXTINF:-1 tvg-id="KOTR-LD.us" tvg-name="11 MyTV (KOTR-LD)" tvg-country="US" tvg-language="English" tvg-logo="https://upload.wikimedia.org/wikipedia/en/b/bd/Kotrlp.jpg" group-title="Local",11 MyTV (KOTR-LD) (480i)
 https://cdnapisec.kaltura.com/p/2630451/sp/2630451/playManifest/entryId/1_hq4vlk40/deliveryProfileId/672/protocol/https/format/applehttp/a.m3u8
-#EXTINF:-1 tvg-id="13thStreet.us" tvg-name="13th Street (Poland)" tvg-country="PL" tvg-language="Polish" tvg-logo="" group-title="",13th Street (Poland)
-http://91.218.60.3:8001/play/a004
```

Comment:
![image](https://user-images.githubusercontent.com/5430116/121781045-b7e4bf80-cbab-11eb-8e1b-39364d10e137.png)

Reply:
But it isn't US that is why I removed it

Reply:
We're following Lyngsat principles...
https://www.lyngsat.com/tvchannels/us/13-Ulica.html

Reply:
Okay, but just to let you know, we can't get East Satellite in the US and only go down to 60s ish in the Western Hemisphere.